rbd/main.go

```diff
@@ -50,6 +50,10 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *metadataStorage != "node" && *metadataStorage != "k8s_configmap" {
```

Comment:
The validation is already done here: 
https://github.com/ceph/ceph-csi/blob/master/pkg/util/cachepersister.go#L38-L50